docs/change-log.md

```diff
@@ -2,6 +2,20 @@
 
 This document provides an overview over the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update February 5th, 2021
+
+- **New:** Added support to provision more than one resource of the same type for each workload.  
+![Multiple Resources of same Type](_assets/images/2021-02-05_Multiple_Resources.png)
+- **New:** Error messages on the workload/replica level now also include deployment errors and not only runtime errors (see update of deployment logic below).
+- **New:** Added support for Aiven (Database-as-a-Service provider).  
+![Added Aiven Support](_assets/images/2021-02-05_Aiven_Support.png)
+
+- **Improved:** Updated deployment logic. Deployments in Humanitec no longer wait for pods to start successfully. Instead, errors related to pulling an image of starting a container are now shown as runtime errors (see update of error messages above).
```

Comment:
This should not be "Improved" it should be "Change". The text should be:

> We changed the default deployment behaviour so that workloads no longer wait for a successful readiness probe before terminating the old pods. This means that the state of the namespace will match the requested deployment, but will mean that pods might be in a failed state. These will be reported in the UI. It will also mean deployments complete much faster. We are working on [Blue/Green Deployments](./roadmap.md#bluegreen-deployment-strategy) to provide a reliable way of ensuring environment wide consistency.

Reply:
Changed